functions/go/set-labels/transformer/const.go

```diff
@@ -0,0 +1,196 @@
+package transformer
+
+const (
+	fnConfigGroup         = "fn.kpt.dev"
+	fnConfigAPIVersion    = "v1alpha1"
+	fnConfigKind          = "SetLabels"
+	fnDeprecateField      = "additionalLabelFields"
+	CommonLabelFieldSpecs = `
```

Comment:
Have we agreed to change this yaml?

The content here doesn't appear to be used anywhere.

Reply:
+1, please cleanup @zyy98